COLLABORATOR_GUIDE.md

```diff
@@ -802,9 +802,8 @@ pulled from the staging branch into the LTS branch only when a release is
 being prepared and may be pulled into the LTS branch in a different order
 than they were landed in staging.
 
-Any Collaborator may land commits into a staging branch, but only the release
-team should land commits into the LTS branch while preparing a new
-LTS release.
+Only the members of the @nodejs/backporters team should land commits onto
```

Comment:
Was into -> onto a typo or done purposely in the last commit?
```suggestion
Only the members of the @nodejs/backporters team should land commits into
```

Reply:
I change this as commits land on a branch... happy to change it back.